modules/navigation/2d/nav_mesh_generator_2d.cpp

```diff
@@ -236,6 +246,15 @@ void NavMeshGenerator2D::generator_parse_geometry_node(Ref<NavigationPolygon> p_
 	generator_parse_tilemap_node(p_navigation_mesh, p_source_geometry_data, p_node);
 	generator_parse_navigationobstacle_node(p_navigation_mesh, p_source_geometry_data, p_node);
 
+	generator_rid_rwlock.read_lock();
+	for (const NavMeshGeometryParser2D *parser : generator_parsers) {
+		if (parser->callback.is_null()) {
```

Comment:
```suggestion
		if (!parser->callback.is_valid()) {
```
`Callable` works differently I believe, needs to be `is_valid`